src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -278,25 +278,37 @@ public override void Dispose()
             this.logger.LogInformation("Flushing peers...");
             this.flushAddressManagerLoop.Dispose();
 
+            this.logger.LogInformation("peerAddressManager...");
```

Comment:
What is the user experience here seeing `peerAddressManager...`? How about writing English sentence? Also don't expose names of classes or variables or properties, write normal English sentence `Disposing partial block validator...`

Reply:
Ok makes sense.

Reply:
I think it is OK to put the name of the class, or something that doesn't come up in the Oxford dictionary, we are really logging info about implementation details, so it is not shocking to me

Reply:
Then it doesn't belong on an information level